drv/fpga-api/src/lib.rs

```diff
@@ -0,0 +1,128 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! API crate for the FPGA server.
+
+#![no_std]
+
+use drv_spi_api::SpiError;
+use userlib::*;
+use zerocopy::AsBytes;
+
+#[derive(Copy, Clone, Debug, PartialEq)]
+#[repr(u16)]
```

Comment:
What does this `#[repr(u16)]` do?

[Testing on Playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=2024a0df8df09c9621e3c645b180ef40), I find that it increases the size of the type from 2 bytes to 4 bytes.

Reply:
Ah also left over from an earlier iteration, removed. Thanks!